### ISSUE TITLE
"Building and Examining Packets"?

### ISSUE THREAD

**Issue by clasnake** [brandon-rhodes/fopnp#12]
Hi Brandon,
In the second last paragraph of the introduction of Chapter 2 - UDP, you mentioned the "Building and Examining Packets" section near the end of Chapter 1, which builds raw ICMP packets and receives an ICMP reply. But I don't see there's such a section. Am I missing something here or do you mean the "Packet Fragmentation" section?
Thanks.

**Comment by brandon-rhodes** [comment]
I think that is an error that I failed to catch! I had originally planned to have Chapter 1 use Scapy to actually build and send IP packets so that readers would understand exactly what is inside a packet, but when I realized that this needed to be a Python 3 book and that Scapy only works under Python 2, I cancelled those plans — but apparently forgot to remove a reference to them from Chapter 2!

I will put this on the list of things to fix if Apress is ever interested in another edition of the book. Thanks!
